mixing/mixpool/mixpool.go

```diff
@@ -236,6 +282,21 @@ func (p *Pool) HaveMessage(query *chainhash.Hash) bool {
 	return ok
 }
 
+// RecentMessage searches the mixing pool and cache of recent messages that have
+// been removed for a message by its hash.
```

Comment:
i would reword this comment. it is not searching for only recent messages that have been removed, but any messages that is currently accepted plus any message that was recently removed and still contained in the lru.

Reply:
maybe (well, definitely) it's just because english is dumb and i could see this being interperted multiple ways, but it's definitely not clearly written in any case.

Reply:
Yeah, I meant it as in that is searches the mixing pool and (then if not found) the cache of recent messages, but I can definitely see what you mean that it's easy to read it the other way.

I agree your wording is clearer.  How about this?

> RecentMessages attempts to find a message by its hash in both the mixing pool that contains accepted messages as well as the cache of recently removed messages.

Reply:
sure, def better

Reply:
Updated.